### DISCUSSION TITLE
How to enable buysell function?

### DISCUSSION THREAD

**Discussion by kgt0205** [GriefPrevention/GriefPrevention#2177] [Q&A]
How to enable buysell function?
I can't find settings in config.yml

**Comment by Jikoo** [comment]
Set a price for buying or selling in the config. No prices = not enabled.

**Reply by kgt0205** [reply]
But I set them all up.

  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 10.0
    ClaimBlocksSellValue: 5.0

**Reply by kgt0205** [reply]
![image](https://github.com/TechFortress/GriefPrevention/assets/69426018/1701066e-c5ca-4f80-8399-6b5580b9dd49)

**Reply by Jikoo** [reply]
Check your logs. When you first attempt to use buy or sell commands, the Vault integration will be set up. If it fails, i.e. if you do not have a Vault-compatible economy installed or some other error, buying and selling will also be disabled.

**Reply by QarthO** [reply]
I learned that you need to restart the server in order for this to apply, using the gp reload command doesn't work after updating the config (only for this specific setting)
